code/index.html

```diff
@@ -1,22 +1,288 @@
+          href="https://www.syracuse.com/business/2022/01/cny-maple-syrup-farm-closes-saying-bittersweet-goodbye-after-45-years.html">
+          <img src="./news-images/law-close.jpg" alt="Close-up of a sugar shack">
+        </a>
+        <a class="card-headline" target="_blank"
+          href="https://www.syracuse.com/business/2022/01/cny-maple-syrup-farm-closes-saying-bittersweet-goodbye-after-45-years.html">Maple
+          syrup farm closes, saying bittersweet goodbye after 45 years</a>
+      </div>
+
+      <div class="card society">
+        <a class="card-picture" target="_blank"
+          href="https://newsd.in/national-maple-syrup-day-2021-history-observance-celebrations-and-traditions-of-this-delicious-day/">
+          <img src="./news-images/society-national.jpg" alt="Bottle of maple syrup">
+        </a>
+        <a class="card-headline" target="_blank"
+          href="https://newsd.in/national-maple-syrup-day-2021-history-observance-celebrations-and-traditions-of-this-delicious-day/">National
+          Maple Syrup Day: celebrations and traditions of this delicious day</a>
+      </div>
+
+      <div class="card law">
+        <a class="card-picture" target="_blank"
+          href="https://nationalpost.com/news/canada/the-great-maple-syrup-heist-trial-opens-in-largest-theft-ever-investigated-by-quebec-police">
+          <img src="./news-images/law-trial.jpg" alt="Bundles of Canadian money">
+        </a>
+        <a class="card-headline" target="_blank"
+          href="https://nationalpost.com/news/canada/the-great-maple-syrup-heist-trial-opens-in-largest-theft-ever-investigated-by-quebec-police">The
+          great maple syrup heist: Trial opens in largest theft ever in Quebec</a>
+      </div>
+
+      <div class="card travel">
+        <a class="card-picture" target="_blank"
+          href="https://news.yahoo.com/park-offers-maple-syrup-field-152938651.html">
+          <img src="./news-images/travel-bus.jpg" alt="School buses">
+        </a>
+        <a class="card-headline" target="_blank"
+          href="https://news.yahoo.com/park-offers-maple-syrup-field-152938651.html">Park offers maple syrup field trip
+          opportunity</a>
+      </div>
+
+      <div class="card society">
+        <a class="card-picture" target="_blank"
+          href="https://www.wweek.com/restaurants/2021/12/29/jim-dixons-diy-dish-swedish-style-caramelized-cabbage/">
+          <img src="./news-images/society-cabbage.jpg" alt="Cooked cabbage">
+        </a>
+        <a class="card-headline" target="_blank"
+          href="https://www.wweek.com/restaurants/2021/12/29/jim-dixons-diy-dish-swedish-style-caramelized-cabbage/">Jim
+          Dixon’s DIY Dish: Swedish-Style Caramelized Cabbage</a >
+      </div>
 
-    <section class="">
-      <!-- The other news  -->
     </section>
-  </body>
-</html>
+    <!-- END News section -->
+  </main>
+
+  <footer>
+    <div class="copyright">
+      <span>Created by Nadia Lefebvre 🐘</span>
```

Comment:
This is the **only** question I have in your code. If you use emojis in HTML, it says in this article that you need to also use aria-labels to make them more accessible for those with screen readers, this is where I found the information: https://www.kirupa.com/html5/emoji.htm
I´m sure we will learn more about accessibility further on.